src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -287,9 +294,16 @@ public int getPendingCompactions() {
   public boolean isRepairRunning() {
     // Check if AntiEntropySession is actually running on the node
     try {
-      ObjectName name = new ObjectName(AES_OBJECT_NAME);
-      int activeCount = (Integer) mbeanServer.getAttribute(name, "ActiveCount");
-      long pendingCount = (Long) mbeanServer.getAttribute(name, "PendingTasks");
+      int activeCount = (Integer) mbeanServer.getAttribute(new ObjectName(AES_ACTIVE_OBJECT_NAME), VALUE_ATTRIBUTE);
+      long pendingCount = (Long) mbeanServer.getAttribute(new ObjectName(AES_PENDING_OBJECT_NAME), VALUE_ATTRIBUTE);
+      
+      try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1141)